test/integration/taskBasedStatusUpdate.test.js

```diff
@@ -537,4 +537,63 @@ describe("Task Based Status Updates", function () {
       );
     });
   });
+
+  describe("PATCH Update User Status on Task Assignment by SuperUser", function () {
+    let userId1;
```

Comment:
Minor comment: As long as you are not initializing these variables, might as well define them in one line.
```
let userId1, user2Name,....;
```